client/my-sites/posts/style.scss

```diff
@@ -1,3 +1,18 @@
+.posts.main {
+	display: block;
+	max-width: 720px;
+
+	@include breakpoint( ">1040px" ) {
+		display: flex;
+		max-width: 1040px;
```

Comment:
I think this is not accounting for `/posts/drafts` screen where drafts list is not shown. Right now the alignment is off, as if it's expecting there to be a draft panel to the right of the main post listing.